examples/prism-gradle.html

```diff
@@ -0,0 +1,56 @@
+<h2>Full example</h2>
+<pre><code>
+    apply plugin: "java"
```

Comment:
Could you please un-indent the code once? It's weird that all lines start with 4 spaces.

Also, please remove the line break after `<code>` and make it `...<code>apply ...`. Right now, the code block starts with an empty line, and that's a little weird.

Reply:
Fixed